uds/addressing/abstract_addressing_information.py

```
@@ -145,7 +145,7 @@ def validate_addressing_params(self, **addressing_params: Any) -> dict[str, Any]
         """Check whether the provided parameters are complete and correct."""
 
     @abstractmethod
-    def is_input_packet(self, **frame_attributes: Any) -> None | AddressingType:  # noqa: vulture
+    def is_input_packet(self, **frame_attributes: Any) -> AddressingType | None:  # noqa: vulture
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**frame_attributes`

(ANN401)